web/public/mgmt/config.yml

```diff
@@ -3664,6 +3653,40 @@ collections:
     folder: "content/src"
     fields:
       - { name: "" }
+  - name: "current-license-status-defaults"
+    label: "Current License Status Defaults"
```

Comment:
dust - personallly not a huge fan of current as a name for variables typically.

Reply:
changed